parser/parser.y

```diff
@@ -2561,6 +2562,10 @@ FunctionCallNonKeyword:
 		args := []ast.ExprNode{$3.(ast.ExprNode), $5.(ast.ExprNode), $7.(ast.ExprNode)}
 		$$ = &ast.FuncCallExpr{FnName: model.NewCIStr($1.(string)), Args: args}
 	}
+|       "REVERSE" '(' Expression ')'
+        {
+                $$ = &ast.FuncCallExpr{FnName: model.NewCIStr($1.(string)), Args: []ast.ExprNode{$3.(ast.ExprNode)}}
+        }
```

Comment:
In parser.y and scanner.y, we use tab to align the code.
Please change spaces to tabs.